Fix test execution harness to actually run the tests, fix data-system-tests, add test-query (#9)
MAAP-Project/maap-data-system-tests: README.md

## GitHub App Creation and Installation

A GitHub App is used as an authentication principal within GitHub that can be granted
fine-grained permissions to perform actions within a repo. This allows clients to perform authorized actions in GitHub without needing to use credentials for a specific user.  

The GitHub App [maap-system-tests](https://github.com/settings/apps/maap-system-tests) has been created
to be used by GitHub Actions in the service repos to invoke the system tests in this repo. 
In this case, a single permission granted is granted to allow invocation of the `workflow_dispatch`

> Action.

Author: @abarciauskas-bgse added this language per your request

The workflow can be manually invoked when testing it using the following steps.

> 0. The App is [maap-system-tests](https://github.com/settings/apps/maap-system-tests) and is part of the MAAP-Project org.

Author: @abarciauskas-bgse updated this

> 0. The App is [maap-system-tests](https://github.com/settings/apps/maap-system-tests) and is part of the MAAP-Project org.

Reviewer: ```suggestion
0. [maap-system-tests](https://github.com/settings/apps/maap-system-tests) is a "github app" of the MAAP-Project org.
```
